.github/workflows/ci.yaml

```diff
@@ -11,10 +11,10 @@ jobs:
       matrix:
         java: [8, 11]
     steps:
+    - uses: actions/checkout@v2
     - uses: stCarolas/setup-maven@v4
       with:
         maven-version: 3.8.1
-    - uses: actions/checkout@v2
```

Comment:
what is a reason to change the order?

Reply:
The order of these commands across libraries is as owlbot's change here. Set up the environment first, then setup the maven version.